### DISCUSSION TITLE
site scam https://erfan4lx.com/

### DISCUSSION THREAD

**Discussion by Lee12367** [community/community#141273] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

https://erfan4lx.com/
this site scame sell program tele
![photo_2024-10-12_21-38-46 (3)](https://github.com/user-attachments/assets/605bf35d-8595-47eb-b492-b9b7a5c77f6e)
![photo_2024-10-12_21-38-46](https://github.com/user-attachments/assets/382bf9be-54ae-48d0-8234-5c73daa2d2e0)
![photo_2024-10-12_21-38-46 (2)](https://github.com/user-attachments/assets/7f2899f2-02dc-4966-b330-a4d866c48058)

**Comment by RudraOp9** [comment]
You can try reporting the website on different platform , so that it could be taken down
[an wikihow article on how to report sites](https://www.wikihow.com/Report-a-Website)

**Comment by ebndev** [comment]
Hi @Lee12367, 👋🏻 The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms however, we are unable to assist with anything outside of the GitHub platform. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
